Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -6,19 +6,6 @@ import QuartzCore
 // MARK: - Aztec Layout Manager
 //
 class LayoutManager: NSLayoutManager {
-
-    /// Blockquote's Left Border Color
-    ///
-    var blockquoteBorderColor = UIColor(red: 0.52, green: 0.65, blue: 0.73, alpha: 1.0)
-
-    /// Blockquote's Background Color
-    ///
-    var blockquoteBackgroundColor = UIColor(red: 0.91, green: 0.94, blue: 0.95, alpha: 1.0)
-
-    /// HTML Pre Background Color
-    ///
-    var preBackgroundColor = UIColor(red: 243.0/255.0, green: 246.0/255.0, blue: 248.0/255.0, alpha: 1.0)
-
```

Comment:
Mind reverting this one sir? (commit `13ed3719b02397821e2186e6ff85bf7e9b03ced6`). Reason is that by having these colors as instance variables, App Devs may run multiple Aztec instances, each one with different Blockquote / Pre Colors.

By moving this to static vars, we loose the ability to tailor down specific instances.